bin/MadGraph5_aMCatNLO/cards/production/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat

```diff
@@ -1,12 +1,7 @@
 import model loop_sm-ckm_no_b_mass
-#switch to diagonal ckm matrix if needed for speed
-#import model loop_sm-no_b_mass
-
-define p = p b b~
-define j = j b b~
 
 generate p p > a a [QCD] @0
 add process p p > a a j [QCD] @1
 add process p p > a a j j [QCD] @2
 
-output diphotonJets_5f_NLO_FXFX_Mgg80ToInf -nojpeg
+output DiphotonJets_5f_NLO_FXFX_Mgg80ToInf -nojpeg
```

Comment:
If you change the output name, you should change the corresponding cards' names as well.
e.g. diphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat -> DiphotonJets_5f_NLO_FXFX_Mgg80ToInf_proc_card.dat